content/community/community-team/community-building-roles/contents.lr

```diff
@@ -13,54 +13,71 @@ Community building roles are for folks interested in helping grow CC's open sour
+**Who should apply:**  
+If you’ve helped anyone else in the CC open source community on GitHub, Slack, or email, you should apply for this role!
 
-**What does this role give you?**
-* You’ll receive invites to the private CC Community Team Slack channel and email list.
+**What does this role give you?**  
 * You’ll receive previews of upcoming changes to the Community Team program.
 * You’ll be listed as [a member of the Community Team on CC Open Source](/community/community-team/members/).
 
+**How can you engage with the community?**  
+You can use the following channels to engage with the community:
+* [Vote access](/community/community-team/engagement-channels/#vote-access)
+  on our upcoming tweets
+* Invitation to our Slack channels
+  * [Newbie channels](/community/community-team/engagement-channels/#newbie-channels)
+  * [UX design channels](/community/community-team/engagement-channels/#ux-design-channel)
+  * [Community Team channel](/community/community-team/engagement-channels/#community-team-channel)
+* Invitation to our [meetings](/community/community-team/engagement-channels/#meetings)
+* Membership in the [Community Team mailing list](/community/community-team/engagement-channels/#community-team-group)
+
 **Guidelines for Community Contributors**  
 If you’ve been accepted as a Community Contributor, you are encouraged to:
 * keep helping people in the CC open source community!
-  * respond to people on Slack
-  * respond to questions or issues on GitHub
-  * respond to people on the CC developer mailing list
+  * Respond to people on Slack.
+  * Respond to questions or issues on GitHub.
+  * Respond to people on CC's mailing lists.
 * participate in discussions in Slack or via email.
-* contribute in other ways if you’d like, like writing code or reviewing pull requests.
+* contribute in other ways if you’d like, like [writing code or reviewing pull requests](/community/community-team/project-roles/).
 
 ## Community Collaborator
-**Who should apply:** If you’ve helped a few people in the CC open source community and you’d like to do more to help grow the community, you should apply for this role!
+**Who should apply:**  
+If you’ve helped a few people in the CC open source community and you’d like to do more to help grow the community, you should apply for this role!
 
-**What does this role give you?**
+**What does this role give you?**  
 * Everything a Community Contributor gets.
-* Comment-only access to the Open Source Community project on Asana.
-  * You’ll see our future plans for the open source community and have the opportunity to participate in them.
+
+**How can you engage with the community?**  
+In addition to the channels afforded to a Community Contributor, you can use the following channels to engage with the community:
+* [Suggest access](/community/community-team/engagement-channels/#suggest-access)
```

Comment:
see above

Reply:
Merged Vote, Suggest into Edit in bde9b54.